frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckReAuthUserHandler.java

```diff
@@ -108,6 +108,8 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                     .flatMap(
                             userProfile -> {
                                 if (hasEnteredIncorrectEmailTooManyTimes(userProfile.getEmail())) {
+                                    if (configurationService.supportReauthSignoutEnabled())
+                                        removeEmailCountLock(userProfile.getEmail());
```

Comment:
```suggestion
                                    {
                                        removeEmailCountLock(userProfile.getEmail());
                                    }
```

Reply:
Do we have tests for enabled and disabled?

Reply:
I don't think it is needed for the following reasons:

1 - The feature switch comes by default false. So in the case the reauth is switch on in an env, the user will just be locked out if the feature switch isn't modified.
2 - We do not still have the full scope for reauth. So until we have more details about what errors and what audit events we should be sending, I propose we wait and write them once :)